package.json

```diff
@@ -43,11 +43,11 @@
   "size-limit": [
     {
       "path": "dist/web-reader.cjs.production.min.js",
-      "limit": "600 KB"
+      "limit": "650 KB"
```

Comment:
I've increased the size limit because adding the pdfjs package made it bigger - wondering if it's possible to only import the pdf or the epub packages based on what you need.